src/localisation/es.json

```diff
@@ -0,0 +1,201 @@
+        "enableInnerDisabledFrameConfirm":"Esta acción habilitará el el grupo completo de marcos deshabilitados al que pertence este marcos.\n¿Aún quieres continuar?",
+        "importsContainer":"Importaciones:",
+        "funcDefsContainer":"Definiciones de funciones:",
+        "mainContainer":"Mi código:",
+        "preCompiledErrorNeedFix":"Primero corrija los erroes existentes.",
+        "noWebUSB":"Este navegador no admite conexiones webUSB. Por favor utilice otro navegador (como Google Chrome).",
+        "emptyCodeError":"El códdigo no puede estar vacío.",
+        "loadToTarget":"Abierto desde",
+        "saveToTarget":"Guardar en",
+        "targetFS":"Este dispositivo",
+        "fileName":"Nombre del archivo:",
+        "gdriveFileAlreadyExists":"Ya existe un proyecto con este nombre en la ubicación de Google Drive.\n/¿Quieres reemplazarlo?",
+        "gdriveLockedFileAlreadyExists":"Este proyecto está bloqueado en la unicación de Google Drive.\nNo se puede guardar con este nombre.",
+        "autosaveGDrive":"Guardardo",
+        "autoSaveGDriveTooltip":"Este proyecto se guardará automáticamente en Google Drive cada {frecuencia} minutos.",
+        "selectFolder":"Seleccione una carpeta",
+        "selectStrypeFile":"Seleccione un proyecto Strype",
+        "gdriveLocation":"Carpeta:",
+        "gdriveAllStrypeFiles":"Todos los archivos Strype",
+        "gdriveTab":"Mi Drive",
+        "editorErrors":"Errores"
+    },
+
+    "contextMenu": {
+        "ctrl":"Ctrl",
+        "cut":"cortar",
+        "copy":"copiar",
+        "downloadAsImg":"descargar como imagen",
+        "duplicate":"Duplicar",
+        "paste":"pegar",
+        "pasteAbove": "pegar arriba",
+        "pasteBelow": "pegar abajo",
+        "pasteBelowJointRoot": "pegar debajo de lar raíz",
+        "enable":"activar",
+        "disable":"desactivar",
+        "undo":"deshacer",
+        "redo":"rehacer",
+        "delete":"eliminar",
+        "deleteOuter":"eliminar externo",
+        "insert":"insertar"
+    },
+
+    "appMenu": {
+        "downloadPython":"Convertir a archivo Python",
+        "loadProject":"Abrir",
+        "loadProjectTooltip":"Abrir proyecto desde este dispositivo o desde Google Drive",
+        "saveProject":"Guardar",
+        "saveProjectTooltip":"Guardar el proyecto en este dipositivo o en Google Drive",
+        "saveAsProject":"Guardar como...",
+        "saveAsProjectTooltip":"Guardar como un proyecto nuevo en este dipositivo o en Google Drive",
+        "resetProject":"Nuevo proyecto",
+        "resetProjectTooltip":"Restablecer el editor con el proyecto predeterminado de Strype",
+        "prefs":"Preferencias",
+        "lang":"Idioma: ",
+        "version":"versión",
+        "homepage":"Página de inicio de Strype"
+    },
+
+    "autoCompletion": {
+        "myFunctions":"Mis funciones ",
```

Comment:
We should probably trim the trailing space.

Reply:
Well spotted